README.md

```diff
@@ -2,83 +2,119 @@
 
 # Elastic Common Schema (ECS)
 
-The Elastic Common Schema (ECS) is used to provide a common data model when
-ingesting data into Elasticsearch. Having a common schema allows you correlate
-data from sources like logs and metrics or IT operations
-analytics and security analytics.
+The Elastic Common Schema (ECS) provides a common data model for
+ingesting data into Elasticsearch. The common schema helps you correlate data from sources like logs and metrics or IT operations analytics and security analytics. 
 
 ECS is still under development and backward compatibility is not guaranteed. Any
 feedback on the general structure, missing fields, or existing fields is appreciated.
 For contributions please read the [Contributing Guide](CONTRIBUTING.md).
 
-The current version of ECS is `0.1.0`.
+<a name="ecs-version"></a>The current version of ECS is `0.1.0`.
 
+## In this readme
+
+* [Implementing ECS](#implementing-ecs)
 * [Fields](#fields)
+  * [Base fields](#base)
```

Comment:
Is the minor change in structure worth the effort to modify the script?  I like the  fields nested in a mini-TOC, but it might not be worth the time/effort to change the script.